platform/npm/tests/request-data.js

```diff
@@ -96,10 +100,6 @@ describe('Request data', () => {
                     read('./data/assets/thirdparties/urlhaus-filter/urlhaus-filter-online.txt')
                         .then(raw => ({ name: 'urlhaus', raw })),
                 ]);
-
```

Comment:
@gorhill it just occurred to me that ideally `enableWASM()` should be called before any other initialization.

Reply:
Yes, this avoids having to re-allocate all the typed arrays used by the JS paths before they are grown.